Sources/Sentry/Public/SentryDefines.h

```diff
@@ -115,18 +115,6 @@ typedef NS_ENUM(NSInteger, SentryPermissionStatus) {
     kSentryPermissionStatusDenied
 };
 
-/**
```

Comment:
`m`: Stricly speaking all changes in `/Public/...` are breaking changes, but no one should be affected by these changes, but you never know.

Reply:
@armcknight, can we keep the old public values and mark them as deprecated? I think that would be safer.

Reply:
Ah, I missed that this was in public headers. Yes, we can try to mark them as deprecated and I'll make a note of it [here](https://github.com/getsentry/sentry-cocoa/discussions/1094#discussioncomment-3588483).